spacemacs/packages.el

```diff
@@ -1242,6 +1242,8 @@ Example: (evil-map visual \"<\" \"<gv\")"
                                          "speedbar-mode"
                                          ))
 
+      (add-to-list 'golden-ratio-exclude-buffer-regexp "*[hH]elm.*")
```

Comment:
Correct, the package owner changed the regex check after I submitted the patch on that side.  
Thanks.